transcribe/render.py

```diff
@@ -50,12 +57,19 @@ def transcribe(input_path, scribe_type):
                 except ImportError as e:
                     print(e)
                     continue
-                schema = load_schema_file(key)
-                try:
-                    output_doc = \
-                        validate_document(dict(scribe_module_instace),
-                                          schema)
                 except ValueError as e:
                     print(e)
                     continue
-                yield(json.dumps(output_doc, indent=4))
+                except TypeError as e:
+                    print(e)
+                    continue
+                schema = load_schema_file(key)
```

Comment:
These lines, 66 - 75, look identical to lines 38 - 47.  Any way to apply the DRY principle here?